packages/mui-material/src/Select/Select.test.js

```diff
@@ -1439,6 +1440,70 @@ describe('<Select />', () => {
     expect(container.getElementsByClassName(classes.select)[0]).to.toHaveComputedStyle(selectStyle);
   });
 
+  it('it should override with error style when `native select` has `error` state', function test() {
```

Comment:
Can you move both the `it` blocks inside a `describe` block? It can have a description as `describe('theme styleOverrides: ')`